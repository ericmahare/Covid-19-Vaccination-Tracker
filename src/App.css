* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

:root {
  --light-blue: #5788e4;
  --dark-blue: #4369b2;
}

body {
  color: #fff;
  background-color: var(--dark-blue)
}