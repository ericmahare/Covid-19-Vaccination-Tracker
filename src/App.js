import { Provider } from 'react-redux';
import './App.css';
import Countries from './components/Countries';
import CountryData from './components/CountryData';
import store from './redux/configurationStore';
import { Routes, Route } from "react-router-dom";

function App() {
  return (
    <Provider store={store}>
      <Routes>
        <Route path="/" element={ <Countries/> } />
        <Route path="/:country_name" element={ <CountryData/> } />
      </Routes>
    </Provider>
  );
}

export default App;
