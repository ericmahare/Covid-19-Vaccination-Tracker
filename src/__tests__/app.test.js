import renderer from 'react-test-renderer';
import App from '../App';
import store from '../redux/configurationStore';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer.create(
      <Provider store={store}>
        <Router>
          <App/>
        </Router>
      </Provider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})