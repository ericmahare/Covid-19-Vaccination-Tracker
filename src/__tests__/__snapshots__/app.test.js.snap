// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <header>
    <nav>
      <ul
        className="nav-list"
      >
        <li>
          2022
        </li>
        <li>
          Covid-19 Vaccination
        </li>
        <li>
          <i
            className="fa-solid fa-microphone"
          />
          <i
            className="fa-solid fa-gear"
          />
        </li>
      </ul>
    </nav>
    <div
      className="hero"
    >
      <div
        className="hero-img-container"
      >
        <img
          alt="Africa"
          className="hero-img"
          src="africa.svg"
        />
      </div>
      <div
        className="hero-info"
      >
        <h2>
          Africa Total Vaccination
        </h2>
        <h2>
          0 people vaccinated
        </h2>
      </div>
    </div>
  </header>,
  <div
    className="headline"
  >
    <h2>
      Vaccination by Country
    </h2>
  </div>,
  <section
    className="countries"
  />,
]
`;
